Source/WebKit/gtk/tutorial.md

```diff
@@ -0,0 +1,382 @@
+# Create a Web Browser in GTK with WebKit
+
+This is a tutorial for creating a simple web browser with some basic functionalities in GTK with WebKit in C language.
+
+GTK is a multi-platform toolkit for creating graphical user interfaces; it contains a complete set of widgets to be reused and combined together in development.
```

Comment:
Probably I would add one more line indicating what WebKitGTK provides on top of GTK. Something like this, maybe:

```markdown
WebKitGTK provides a [class@WebKitWebView] widget which can load and display
web content, which allows integrating web technologies (HTML, CSS, JavaScript)
in your applications&mdash;all the way to implement full-fledged browsers like
[GNOME Web](https://wiki.gnome.org/Apps/Web/).
```

Reply:
Maybe we can also link to [the GTK “getting started”](https://docs.gtk.org/gtk3/getting_started.html) page, mentioning that it is recommended to go through it first to learn about the basics of using GTK for those people who may need it.